pkg/activator/net/throttler_test.go

```diff
@@ -640,16 +640,23 @@ func TestActivatorsIndexUpdate(t *testing.T) {
 		t.Fatalf("RevisionThrottler can't be found: %v", err)
 	}
 
-	// Verify the index was computed.
+	// Verify capacity gets updated. This is the very last thing we update
+	// so we now know that the rest is set statically.
 	if err := wait.PollImmediate(10*time.Millisecond, time.Second, func() (bool, error) {
-		return atomic.LoadInt32(&rt.numActivators) == 2 &&
-			atomic.LoadInt32(&rt.activatorIndex) == 1, nil
+		// Capacity doesn't exceed 1 in this test.
+		return rt.breaker.Capacity() == 1, nil
```

Comment:
@vagababov this seems actually wrong. Let's discuss the behavior here.

Reply:
Once public eps are created the informer will fire and activator will recompute. But yeah I can see that the actual update won't finish by then. 
Capacity should be 2. 3 endpoints 2 activators